### ISSUE TITLE
Status of the package?

### ISSUE THREAD

**Issue by randomor** [KjellConnelly/react-native-cloudkit#1]
Hi @KjellConnelly, 

Thanks for making this package! I'm wondering if you are still making progress on this package? I've tried CloudKit.js V2 with `react-native-crypto`, but I was blocked at the `container.setupAuth` step, and I'm wondering if its because it's requiring a browser. I'm considering whether I should use this vs wrapping a native module at this point. Would appreciate any updates!

**Comment by KjellConnelly** [comment]
Hey randomor! You're very welcome! I haven't been working on it lately as it's not on my priority list of things to add to the current app's I'm working on. Unfortunately, I'm not going to work on this for awhile. But probably will in maybe the next 6 months to implement cloud syncing for my own users. 

I tried v2 and there were issues for me, and since v1 worked for me, I didn't bother with v2 after that. I'm not sure of the ramifications, but I'm also not building extremely professional apps either. So I'm fine with v1. 

Not sure what you're going to be using CloudKit for specifically, but I was able to do some basic things via a public database with it. If you go through the documentation, you could probably do just about everything using this module because if I remember correctly, it returns the v1 CloudKit object when you start it up. Of course, it's been a few months since I used this module, and I only messed around with it for a day or 2. You'd have to look at the actual JavaScript documentation to see what it can and cannot do.

**Comment by randomor** [comment]
Thanks for the prompt response! I think the issue must be the "public" database vs "private" one. I was able to make queries to the public database with the v2. But with private database, I have to go through the OAuth authorization flow, and even if I were able to get the user token eventually (if I read more about the web service docs first), CloudKit.js doesn't have a way to accept it, and the flow also doesn't make sense because the user is already logged in on their iPhones... I guess my next step has to be the native module wrapper... Thanks again for the ideas and good luck with your app!
